Assets/Scripts/AudioScript.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioScript : MonoBehaviour
+{
+    [SerializeField] private AudioSource audioPlayer;
+    [SerializeField] private AudioSource musicPlayer;
```

Comment:
So audioPlayer is for narrator and musicPlayer for the background music? If yes, then maybe audioPlayer could be renamed to e.g. `narrativePlayer`, `narrativeAudioPlayer` or something like that to be more clear. Just that I had to find the message in messenger to understand what is meant here 😅